app/controllers/users_controller.rb

```diff
@@ -12,6 +12,7 @@ def create
     @user = User.new(user_params)
     if @user.save
       session[:user_id] = @user.id
+      @user.send_welcome_email
```

Comment:
@matthewrpacker If the creation of a user starts to involve more than saving and sending an email, I might consider creating a PORO to handle this logic... Something like `UserCreator`. That would be overkill right now IMO but is a nice trick to keep in mind.